daemon/src/projection.rs

```diff
@@ -271,7 +271,12 @@ impl Aggregated {
                     Role::Maker => CfdState::IncomingRolloverProposal,
                     Role::Taker => CfdState::OutgoingRolloverProposal,
                 },
-                ProtocolNegotiationState::Accepted => CfdState::ContractSetup,
+                // TODO: we should have an own state here indicating that the position is being
+                // rolled over.
+                ProtocolNegotiationState::Accepted => match role {
+                    Role::Maker => CfdState::IncomingRolloverProposal,
+                    Role::Taker => CfdState::OutgoingRolloverProposal,
```

Comment:
This will have the side effect that we emit the actions again on the maker. We cache emitted actions in the automation so this should be safe for now but can become very brittle very soon. Please amend the commit message to explain why this is ok!

I'd prefer if we can go into the direction of https://github.com/itchysats/itchysats/issues/1314.